mne/viz/_mpl_figure.py

```diff
@@ -75,14 +74,6 @@
 )
 
 name = "matplotlib"
-plt.ion()
-BACKEND = get_backend()
-#   This  ↑↑↑↑↑↑↑↑↑↑↑↑↑  does weird things:
-#   https://github.com/matplotlib/matplotlib/issues/23298
-#   but wrapping it in ion() context makes it go away (can't actually use
-#   `with plt.ion()` as context manager, though, for compat reasons).
-#   Moving this bit to a separate function in ../../fixes.py doesn't work.
-plt.ioff()
```

Comment:
I think the right thing to do here would be to take advantage of the context-manager capabilities (available in [3.4](https://matplotlib.org/3.4.3/api/_as_gen/matplotlib.pyplot.ioff.html) already which is our min req) like:
```
with plt.ion():
    BACKEND = get_backend()
```

Reply:
... and if you could please do this in `mne/viz/backends/_notebook.py` as well, it would be much appreciated! (Found via `git grep` when looking into this issue in our repo...)